mage_ai/server/api/triggers.py

```diff
@@ -44,10 +48,27 @@ def post(self, pipeline_schedule_id, token: str = None):
                 payload['event_variables'][k] = v
 
         pipeline = Pipeline.get(pipeline_schedule.pipeline_uuid)
-        pipeline_run = create_and_start_pipeline_run(
-            pipeline,
-            pipeline_schedule,
-            payload,
-        )
+
+        pipeline_runs = PipelineRun.active_runs_for_pipelines([pipeline.uuid])
```

Comment:
We can simplify the logic here. The API trigger only needs to create the pipeline run and doesn't need to start the pipeline run like https://github.com/mage-ai/mage-ai/blob/master/mage_ai/api/resources/PipelineRunResource.py#L262

The pipeline run start or cancellation logic can be centralized in the `schedule_all` method

Reply:
ah that's smart. i'll change it